### ISSUE TITLE
applyConverter missing?

### ISSUE THREAD

**Issue by allmondjoy87** [terraformer-js/terraformer#68]
How do I use the applyConverter tool with the latest release?

**Comment by jgravois** [comment]
> is this something i can do? i've never contributed to a project before, unsure what the rules are.

absolutely. to do this you'd just need to:

1. clone the repo 
1. run `npm install && npm test` locally to ensure all is well
1. add the relevant `export` in `packages/spatial/src/index.js`
1. supplement that export with the JSDoc below
1. run `npm run doc` to update the README for the spatial package to mention the new method
1. run `npm test` again to confirm all is _still_ well
3. commit the result in your own branch and open up a PR

```js
/**
 * Runs the passed function against every Coordinate in the geojson object.
 * @function
 * @param {object} GeoJSON - The input [GeoJSON](https://tools.ietf.org/html/rfc7946) Geometry, Feature, GeometryCollection or FeatureCollection.
 * @param {function} function - Your function will be passed a Coordinate and will be expected to return a Coordinate.
 * @return {object} GeoJSON - [GeoJSON](https://tools.ietf.org/html/rfc7946) with altered coordinates.
 * ```js
 * import { applyConverter } from "@terraformer/spatial"
 *
 * applyConverter({
 *   type: "Point",
 *   coordinates: [ 45, 60 ]
 * }, (coord) => [coord[0] + 1, coord[1] - 1])
 *
 * >> { type: "Point", coordinates: [ 46, 59 ] }
 * ```
 */
```

**Comment by jgravois** [comment]
if you use Node.js 16x or 17x, the install should succeed.

`patch-68` can't be merged as is. it has a bunch of unrelated whitespace changes.

**Comment by allmondjoy87** [comment]
```bash
node --version
v16.14.0
```
Failing w/ 16x. `npm cache verify` and `npm cache clean --force` didn't help either.

I'll update to 17x and give it a shot.

**Comment by jgravois** [comment]
> Well I'm having issues with npm-install

i _think_ what i did in will fix your issue.

https://github.com/terraformer-js/terraformer/pull/87/files#diff-a2fe61aa17681ca122cfdd06603e7462426b0672cea3517deefdd96b6d4e3523R13

**Comment by jgravois** [comment]
released in [`v2.1.2`](https://www.npmjs.com/package/@terraformer/spatial/v/2.1.2).
